index.html

```diff
@@ -50,10 +50,12 @@ <h1 id="title">freeCodeCamp Survey Form</h1>
                 <option value="val3">Projects</option>
                 <option value="val4">Community</option>
                 <option value="val5">Open Source</option>
+

```

Comment:
- [ ] [OPTIONAL] To encourage the student, it could be better if you by highlighting the effort of the student 😃 
- [ ] [OPTIONAL] Also, it could be better to add `emojis` to make your comment more freindly 👍